website/docs/docs/deploy/job-notifications.md

```diff
@@ -56,7 +56,7 @@ If there has been a change in user roles or Slack permissions where you no longe
 :::
 
 ### Prerequisites 
-- You must be an administrator of the Slack workspace. 
+- You must be a Slack Workspace Owner. 
```

Comment:
Hiya @matthewshaver 

This is more of a question - in the docs "workspace" isn't capitalised and "owner" is capitalised so should "Slack Workspace Owner" be "Slack workspace Owner"?

Kind Regards
Natalie